src/python/tests/ops_simnoise.py

```diff
@@ -1,25 +1,17 @@
 # Copyright (c) 2015-2017 by the parties listed in the AUTHORS file.
-# All rights reserved.  Use of this source code is governed by 
+# All rights reserved.  Use of this source code is governed by
 # a BSD-style license that can be found in the LICENSE file.
 
-from ..mpi import MPI
-from .mpi import MPITestCase
-
-import sys
 import os
 
 import numpy as np
-import numpy.testing as nt
-
-import scipy.interpolate as si
 
-from ..tod.tod import *
-from ..tod.pointing import *
-from ..tod.noise import *
-from ..tod.sim_noise import *
-from ..tod.sim_det_noise import *
-from ..tod.sim_tod import *
```

Comment:
I basically only used wildcard imports inside the unittest source files.  Previously the unittests were external from the package, and so it seemed fine to sacrifice ultra-picky standards in the name of convenience.  However, we have now moved the unit tests into an internal sub-package.  Since this code is now part of the package itself, it does seem to make sense to enforce the normal best practices (i.e. no wildcard imports at all).